src/components/CandidateStatus.js

```diff
@@ -0,0 +1,95 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+
+
+const CandidateStatus = ({ candidate, order }) => {
+  const { id, name, voteCount } = candidate;
+  const [vote, setVote] = useState(null);
+
+  const onClickVote = () => {
+    const fetchData = async () => {
+      try {
+        const response = await axios.get(
+          `http://ec2-3-34-5-220.ap-northeast-2.compute.amazonaws.com:2020/vote?id=${id}`
+        );
+      } catch (e) {
+        console.log(e);
+        alert(name + '님에게 투표실패ㅠㅠ');
+      }
+      alert(name + '님에게 투표성공!');
+    };
+    fetchData();
+  };
```

Comment:
저도 피드백 받은 건데, 특별한 이유가 없다면 함수를 중첩할 필요가 없다고 해서, 이렇게 고치는 것은 어떨까요?!

```suggestion
  const onClickVote = async () => {
      try {
        await axios.get(
          `http://ec2-3-34-5-220.ap-northeast-2.compute.amazonaws.com:2020/vote?id=${id}`
        );
      } catch (e) {
        alert(name + '님에게 투표실패ㅠㅠ');
      }
      alert(name + '님에게 투표성공!');
};

```